BinaryEncoding.md

```diff
@@ -1,122 +1,400 @@
+| `i32.gt_u` | `0x55` | | |
+| `i32.ge_u` | `0x56` | | |
+| `i32.clz` | `0x57` | | |
+| `i32.ctz` | `0x58` | | |
+| `i32.popcnt` | `0x59` | | |
+| `bool.not` | `0x5a` | | |
+| `i64.add` | `0x5b` | | |
+| `i64.sub` | `0x5c` | | |
+| `i64.mul` | `0x5d` | | |
+| `i64.div_s` | `0x5e` | | |
+| `i64.div_u` | `0x5f` | | |
+| `i64.rem_s` | `0x60` | | |
+| `i64.rem_u` | `0x61` | | |
+| `i64.and` | `0x62` | | |
+| `i64.ior` | `0x63` | | |
+| `i64.xor` | `0x64` | | |
+| `i64.shl` | `0x65` | | |
+| `i64.shr_u` | `0x66` | | |
+| `i64.shr_s` | `0x67` | | |
+| `i64.eq` | `0x68` | | |
+| `i64.ne` | `0x69` | | |
+| `i64.lt_s` | `0x6a` | | |
+| `i64.le_s` | `0x6b` | | |
+| `i64.lt_u` | `0x6c` | | |
+| `i64.le_u` | `0x6d` | | |
+| `i64.gt_s` | `0x6e` | | |
+| `i64.ge_s` | `0x6f` | | |
+| `i64.gt_u` | `0x70` | | |
+| `i64.ge_u` | `0x71` | | |
+| `i64.clz` | `0x72` | | |
+| `i64.ctz` | `0x73` | | |
+| `i64.popcnt` | `0x74` | | |
+| `f32.add` | `0x75` | | |
+| `f32.sub` | `0x76` | | |
+| `f32.mul` | `0x77` | | |
+| `f32.div` | `0x78` | | |
+| `f32.min` | `0x79` | | |
+| `f32.max` | `0x7a` | | |
+| `f32.abs` | `0x7b` | | |
+| `f32.neg` | `0x7c` | | |
+| `f32.copysign` | `0x7d` | | |
+| `f32.ceil` | `0x7e` | | |
+| `f32.floor` | `0x7f` | | |
+| `f32.trunc` | `0x80` | | |
+| `f32.nearestint` | `0x81` | | |
+| `f32.sqrt` | `0x82` | | |
+| `f32.eq` | `0x83` | | |
+| `f32.ne` | `0x84` | | |
+| `f32.lt` | `0x85` | | |
+| `f32.le` | `0x86` | | |
+| `f32.gt` | `0x87` | | |
+| `f32.ge` | `0x88` | | |
+| `f64.add` | `0x89` | | |
+| `f64.sub` | `0x8a` | | |
+| `f64.mul` | `0x8b` | | |
+| `f64.div` | `0x8c` | | |
+| `f64.min` | `0x8d` | | |
+| `f64.max` | `0x8e` | | |
+| `f64.abs` | `0x8f` | | |
+| `f64.neg` | `0x90` | | |
+| `f64.copysign` | `0x91` | | |
+| `f64.ceil` | `0x92` | | |
+| `f64.floor` | `0x93` | | |
+| `f64.trunc` | `0x94` | | |
+| `f64.nearestint` | `0x95` | | |
+| `f64.sqrt` | `0x96` | | |
+| `f64.eq` | `0x97` | | |
+| `f64.ne` | `0x98` | | |
+| `f64.lt` | `0x99` | | |
+| `f64.le` | `0x9a` | | |
+| `f64.gt` | `0x9b` | | |
+| `f64.ge` | `0x9c` | | |
+| `i32.sconvertf32` | `0x9d` | | |
+| `i32.sconvertf64` | `0x9e` | | |
+| `i32.uconvertf32` | `0x9f` | | |
+| `i32.uconvertf64` | `0xa0` | | |
+| `i32.converti64` | `0xa1` | | |
+| `i64.sconvertf32` | `0xa2` | | |
+| `i64.sconvertf64` | `0xa3` | | |
+| `i64.uconvertf32` | `0xa4` | | |
+| `i64.uconvertf64` | `0xa5` | | |
+| `i64.sconverti32` | `0xa6` | | |
+| `i64.uconverti32` | `0xa7` | | |
+| `f32.sconverti32` | `0xa8` | | |
+| `f32.uconverti32` | `0xa9` | | |
+| `f32.sconverti64` | `0xaa` | | |
+| `f32.uconverti64` | `0xab` | | |
+| `f32.convertf64` | `0xac` | | |
+| `f32.reinterpreti32` | `0xad` | | |
+| `f64.sconverti32` | `0xae` | | |
+| `f64.uconverti32` | `0xaf` | | |
+| `f64.sconverti64` | `0xb0` | | |
+| `f64.uconverti64` | `0xb1` | | |
+| `f64.convertf32` | `0xb2` | | |
+| `f64.reinterpreti64` | `0xb3` | | |
+| `i32.reinterpretf32` | `0xb4` | | |
+| `i64.reinterpretf64` | `0xb5` | | |
```

Comment:
I think you should use the groups names for all these rather than the v8 names. There has been a lot of confusion in this area. Here's my table:
```
    (i32.trunc_s/f32     #x9d  (f32) nil              i32) ; i32.sconvert_f32
    (i32.trunc_s/f64     #x9e  (f64) nil              i32) ; i32.sconvert_f64
    (i32.trunc_u/f32     #x9f  (f32) nil              i32) ; i32.uconvert_f32
    (i32.trunc_u/f64     #xa0  (f64) nil              i32) ; i32.uconvert_f64
    (i32.wrap/i64        #xa1  (i64) nil              i32) ; i32.convert.i64
    (i64.trunc_s/f32     #xa2  (f32) nil              i64) ; i64.sconvert_f32
    (i64.trunc_s/f64     #xa3  (f64) nil              i64) ; i64.sconvert_f64
    (i64.trunc_u/f32     #xa4  (f32) nil              i64) ; i64.uconvert_f32
    (i64.trunc_u/f64     #xa5  (f64) nil              i64) ; i64.uconvert_f64
    (i64.extend_s/i32    #xa6  (i32) nil              i64) ; i64.sconvert_i32
    (i64.extend_u/i32    #xa7  (i32) nil              i64) ; i64.uconvert_i32
    (f32.convert_s/i32   #xa8  (i32) nil              f32) ; f32.sconvert_i32
    (f32.convert_u/i32   #xa9  (i32) nil              f32) ; f32.uconvert_i32
    (f32.convert_s/i64   #xaa  (i64) nil              f32) ; f32.sconvert_i64
    (f32.convert_u/i64   #xab  (i64) nil              f32) ; f32.uconvert_i64
    (f32.demote/f64      #xac  (f64) nil              f32) ; f32.convert_f64
    (f32.reinterpret/i32 #xad  (i32) nil              f32) ; f32.reinterpret_i32
    (f64.convert_s/i32   #xae  (i32) nil              f64) ; f64.sconvert_i32
    (f64.convert_u/i32   #xaf  (i32) nil              f64) ; f64.uconvert_i32
    (f64.convert_s/i64   #xb0  (i64) nil              f64) ; f64.sconvert_i64
    (f64.convert_u/i64   #xb1  (i64) nil              f64) ; f64.uconvert_i64
    (f64.promote/f32     #xb2  (f32) nil              f64) ; f64.convert_f32
    (f64.reinterpret/i64 #xb3  (i64) nil              f64) ; f64.reinterpret_i64
    (i32.reinterpret/f32 #xb4  (f32) nil              i32) ; i32.reinterpret_f32
    (i64.reinterpret/f64 #xb5  (f64) nil              i64) ; i64.reinterpret_f64
```